validation/hooks_stdin.go

```diff
@@ -42,6 +42,22 @@ func stdinStateCheck(outputDir, hookName string, expectedState rspecs.State) err
 	if !reflect.DeepEqual(state.Annotations, expectedState.Annotations) {
 		return fmt.Errorf("wrong annotations \"%v\" in the stdin of %s hook, expected \"%v\"", state.Annotations, hookName, expectedState.Annotations)
 	}
+
+	switch hookName {
+	case "prestart":
+		if state.Status != "created" {
+			return fmt.Errorf("wrong state %q in the stdin of %s hook, expected %q", state.Status, hookName, "created")
+		}
+	case "poststart":
+		if state.Status != "running" {
+			return fmt.Errorf("wrong state %q in the stdin of %s hook, expected %q", state.Status, hookName, "running")
+		}
+	case "poststop":
+		// Not defined by runtime spec
```

Comment:
I think the current spec wants this to be `stopped` (more discussion in opencontainers/runtime-spec#958), although more spec clarity would be nice.  But what the spec is clear about is that [`status` must be set to *something*][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.1/runtime.md#L16

Reply:
I'll update the code since `'status' must be set to something code.` and wait for runtime-spec#958.